src/components/results/_pagination.scss

```diff
@@ -38,81 +38,81 @@
   @include breakpoint($bp2) {
     margin-right: rhythm(.25);
   }
+}
 
-  // Link styles
-  a {
-    @include trans();
-    @include adjust-font-size-to(.9em, 1);
-    font-weight: 700;
-    display: flex;
-    justify-content: center;
-    align-items: center;
-    border: 0;
-    text-align: center;
-    padding: rhythm(.25);
-
-    @include breakpoint($bp0) {
-      background-color: $c-secondary;
-      color: $white;
-      border-radius: 3px;
-      width: rhythm(1.5);
-      height: rhythm(1.5);
-      padding: 0;
-    }
+// Active page
+.search-pager__item.is-active .search-pager__item-link {
+  @include adjust-font-size-to($text, 1);
 
-    @include breakpoint($bp2) {
-      @include adjust-font-size-to($label, .9);
-      width: rhythm(1.2);
-      height: rhythm(1.2);
-    }
+  @include breakpoint($bp0) {
+    background-color: $link-color;
+    color: $white;
 
     &:hover,
     &:active {
-      text-decoration: underline;
-      color: $white;
-      background-color: $c-secondary;
-
-      @include breakpoint($bp0) {
-        @include adjust-font-size-to($text, 1);
-        background-color: $c-secondary-dark;
-      }
-
-      svg {
-        fill: $c-primary-dark;
-      }
+      background-color: $c-primary-dark;
     }
+  }
+}
 
-    svg {
-      height: 1.3em;
-      fill: $white;
-      display: block;
+.search-pager__item.not-active .search-pager__item-link {
+  display: none;
```

Comment:
@nstriedinger can you walk me through this change? Is this just a result from copy/pasting?

Reply:
@jesconstantine this is just like the original file. I just tried to organize it so it would be easy to grok through.